content/uc-doc/administration/call_recording/index.md

```diff
@@ -58,9 +50,176 @@ To disable online call recording (default: `*3`):
+        {
+          "uuid": "5ea37d0b-0823-4d2b-a18c-b9082e551b50",
+          "start_time": "2021-05-14T17:47:16.701984+00:00",
+          "end_time": "2021-05-14T17:47:22.979637+00:00",
+          "deleted": false,
+          "filename": "2021-05-14T17_47_16UTC-2060-5ea37d0b-0823-4d2b-a18c-b9082e551b50.wav"
+        }
+      ]
+    }
+  ],
+  "total": 887,
+  "filtered": 887
+}
+```
+
+All of the call's information can be viewed, including the ID of the CDR, `2060` here, and the UUID
+of the recording, `5ea37d0b-0823-4d2b-a18c-b9082e551b50`. There can be multiple recordings on a
+single call depending on the call and the configuration.
+
+The curl command to download this recording would be the following.
+
+```bash
+curl -X GET --header 'Accept: audio/wav' --header 'X-Auth-Token: <ACCESS TOKEN>' 'https://<WAZO STACK HOSTNAME>/api/call-logd/1.0/cdr/2060/recordings/5ea37d0b-0823-4d2b-a18c-b9082e551b50/media' --output <MY FILE.wav>
+```
+
+#### Bulk downloads
+
+If you want to export a large selection of recordings to be consumed externally, you can use the
+bulk download API.
+
+First you need to create an export, see the
+[API documentation](https://wazo-platform.org/documentation/console/cdr) for all the available
+options for the export.
+
+```bash
+curl -X POST --header 'Content-Type: application/json' --header 'Accept: application/json' --header 'Wazo-Tenant: <TENANT UUID>' --header 'X-Auth-Token: ACCESS TOKEN' -d '{}' 'https://<WAZO STACK HOSTNAME>/api/call-logd/1.0/cdr/recordings/media/export?from=2021-04-01T00%3A00%3A00-0500&until=2021-05-01T00%3A00%3A00-0500&recurse=false&email=<USERNAME%40DOMAIN>'
+```
+
+The result will contain the UUID of your export. You can use that UUID to download the export or
+query its progression.
+
+To known the status of you export you can use the following command
+
+```bash
+curl -X GET --header 'Accept: application/json' --header 'X-Auth-Token: <ACCESS TOKEN>' 'https://<WAZO STACK HOSTNAME>/api/call-logd/1.0/exports/<EXPORT UUID>'
+```
+
+One the export reaches the status `finished`, mail will be sent to the email address that was in the
+query string of the POST with the link that can be used to download the archive.
+
+#### Automating an Export
+
+Let's say you want to get your recordings exported every week. Here's how you would do it.
+
+1. Create a user with the appropriate permissions to launch the export.
+
+   ```bash
+   export USER_UUID=$(wazo-auth-cli user create --password <password> <username>)
+   export POLICY_UUID=$(wazo-auth-cli policy create --acl call-logd.cdr.recordings.media.export.create auth.tenants.read -- recording_exporter)
+   wazo-auth-cli user add --policy ${POLICY_UUID} ${USER_UUID}
+   ```
+
+2. Create a refresh token that will be copied in the script
+
+   ```bash
+   wazo-auth-cli -vvv token create --access_type offline --client_id exporter --auth-username <username> --auth-password <password> 2>&1 | grep "'refresh_token'" | sed "s/'/\"/g" | jq .refresh_token
```

Comment:
Only one `-v` should be enough
We need to mention to install `jq` which is not installed by default on stack
Maybe we should use a plain grep?